bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java

```diff
@@ -52,6 +52,7 @@
  * Manages a pool of reusable resource resolvers and injects them into tasks
  */
 class ActionManagerImpl extends CancelHandler implements ActionManager, Serializable {
+
     private static final long serialVersionUID = 7526472295622776150L;
 
     transient private static final Logger LOG = LoggerFactory.getLogger(ActionManagerImpl.class);
```

Comment:
The reason I touched ActionManagerImpl in this PR was to change the order of when it does work vs. when it updates its "added" counter.  That makes a difference in how well the junit test performs.  It has no real effect on the real operation since it's async anyway.